proxy/src/main/java/net/md_5/bungee/BungeeCord.java

```diff
@@ -453,7 +467,6 @@ private void independentThreadStop(final String reason)
         saveThread.cancel();
         metricsThread.cancel();
 
-        // TODO: Fix this shit
```

Comment:
This was added in 2013 - seems unlikely it actually needs to be fixed at this point.